### DISCUSSION TITLE
System requirements

### DISCUSSION THREAD

**Discussion by lonix1** [ovh/cds#6278] [Q&A] [answered]
I looked through the docs but didn't find anything related to "system requirements".

I'm sure it depends on how it is configured. But there should be minimum requirements, specifically for memory.

How large a VPS (memory, disk) does one need to run this - without any fancy plugins and addons, the bare minimum?

**Comment by yesnault** [comment, marked as answer]
You will find some details here : https://ovh.github.io/cds/hosting/cds_services/
